rpc/jsonrpc/client/ws_client.go

```diff
@@ -45,8 +45,12 @@ type WSClient struct {
 	// client is being stopped.
 	ResponsesCh chan types.RPCResponse
 
+	// Single user facing channel to get error from, closed only when the
+	// client is being stopped.
+	ErrorCh chan error
```

Comment:
Who's supposed to read from `ErrorCh`? The client?

Reply:
yes, same as ResponsesCh